platforms/Windows/python/python.wxs

```diff
@@ -0,0 +1,159 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Include xmlns="http://wixtoolset.org/schemas/v4/wxs">
+
+  <Package
+      Language="1033"
+      Manufacturer="!(loc.ManufacturerName)"
+      Name="$(VariantProductName)"
+      UpgradeCode="$(VariantUpgradeCode)"
+      Version="$(NonSemVerProductVersion)"
+      Scope="$(PackageScope)">
```

Comment:
How do we version this component? Should we version it as per the Python version or the Swift toolchain version?

Reply:
I think we should version it per the Python version, if we don't change the Python version between Swift versions, there is no need to reinstall Python.